src/components/Executions/filters/useUserByFilterState.ts

```diff
@@ -0,0 +1,53 @@
+import { FilterOperationName } from 'models/AdminEntity/types';
+import { useEffect, useState, useContext } from 'react';
+import { FilterByUserState } from './types';
+import { useFilterButtonState } from './useFilterButtonState';
+import { APIContext } from 'components/data/apiContext';
+
+interface FilterByUserStateArgs {
+    queryStateKey: string;
+    filterOperation?: FilterOperationName;
```

Comment:
I don't think we need a filter operation to be passed. The filter we are generating is always going to use `FilterOperation.EQ`.